src/atomsbase.jl

```diff
@@ -11,17 +10,20 @@ Construct an AtomsBase `FlexibleSystem` from a Chemfiles Frame.
 function AtomsBase.FlexibleSystem(frame::Chemfiles.Frame)
     atoms = map(1:length(frame), frame) do i, atom
         pos = Chemfiles.positions(frame)[:, i]u"Å"
+        velocity_arg = ()
         if Chemfiles.has_velocities(frame)
-            velocity = Chemfiles.velocities(frame)[:, i]u"Å/ps"
-        else
-            velocity = zeros(3)u"Å/ps"
+            velocity_arg = (Chemfiles.velocities(frame)[:, i]u"Å/ps", )
+        end
+
+        species = AtomsBase.ChemicalSpecies(Chemfiles.atomic_number(atom))
+        if Symbol(species) != Symbol(Chemfiles.name(atom))
+            @warn "Ignoring non-standard atom name $(Chemfiles.name(atom)) for atom $i."
```

Comment:
should this check on the atom type instead? Atom name could be something like `CA`, while atom type would be `C`

https://chemfiles.org/Chemfiles.jl/latest/reference/atom/#Chemfiles.type-Tuple{Chemfiles.Atom}